kobuki_velocity_smoother/include/kobuki_velocity_smoother/velocity_smoother.hpp

```diff
@@ -0,0 +1,104 @@
+/**
+ * @file /include/yocs_velocity_smoother/velocity_smoother_nodelet.hpp
+ *
+ * @brief Velocity smoother interface
+ *
+ * License: BSD
+ *   https://raw.github.com/yujinrobot/yujin_ocs/hydro/yocs_velocity_smoother/LICENSE
```

Comment:
Licenses ... this repo generally handles it by copy pasting the content up front here. Should follow that.

Reply:
Fixed in ad27a3a